spec/acceptance/suites/rpm_docker/00_rpm_build_spec.rb

```diff
@@ -0,0 +1,120 @@
+# Uses Docker to do a full build of all SIMP packages
+#
+# Also works in Travis CI
+#
+# If you have a local 'ISO' directory in 'simp-core' with the necessary ISO
+# images, will attempt to build a full SIMP ISO release.
+#
+# If you set the environment variable BEAKER_copyin=yes will copy in your
+# 'simp-core' repo instead of using the one on the filesystem. You probably
+# want to also set BEAKER_destroy=no if you do this so that you can retrieve
+# any relevant artifacts.
+#
+#   * This mode could be **MUCH** slower but will preserve the sanctity of your
+#     workspace
+#
+require 'spec_helper_rpm'
+
+test_name 'RPM build'
+
+# Set build directory inside the container
+#
+# The /simp-core mount will be mounted regardless of the test requirements so
+# we have to use a different location if we want to copy things into the system
+#
+build_dir = ( ENV['BEAKER_copyin'] == 'yes' ) ? '/simp-import': '/simp-core'
+
+# Custom Gemfile Support
+gemfile = nil
+gemfile_path = File.expand_path(File.join(fixtures_path,'Gemfile'))
+
+if File.file?(gemfile_path)
+  gemfile = gemfile_path
+end
+
+describe 'RPM build' do
+  let(:local_basedir) { File.absolute_path(Dir.pwd) }
+
+  # We need a normal user for building the RPMs
+  let(:build_user) { 'build_user' }
+  let(:run_cmd) { %(runuser #{build_user} -l -c ) }
+
+
+  hosts.each do |host|
+    next if host[:roles].include?('disabled')
+
+    # This needs to happen *prior* to the test selection since the test
+    # selection depends on this being present
+    if ENV['BEAKER_copyin'] == 'yes'
```

Comment:
If it is a copyin should it be copying the results over to simp-core so they exist outside the container?